AnnoDesigner/Helper/RoadSearchHelper.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AnnoDesigner.Core.Models;
+
+namespace AnnoDesigner.Helper
+{
+    public static class RoadSearchHelper
+    {
+        private static void DoNothing(AnnoObject objectInRange) { }
```

Comment:
We should use a LayoutObject, not an AnnoObject directly.

Reply:
RoadSearchHelper has strictly backend logic and passing to it frontend class doesn't make sense.